login/profile.go

```diff
@@ -15,6 +15,7 @@ import (
 const ImageURLAttributeName = "ImageURL"
 const BioAttributeName = "Bio"
 const URLAttributeName = "URL"
+const CompanyAttributeName = "Company"
```

Comment:
For `company` fixed it here https://github.com/almighty/almighty-core/pull/1154/commits/1679e781b07639e488ba1fdbf02c7031234b83dd

For the rest, tracking it here :
https://github.com/almighty/almighty-core/issues/1159